### PR TITLE
pkg: enable object property newline rule

### PR DESCRIPTION
**Pull request by jelly**
This is more readable and automatically applied using `eslint --fix`.

### CODE REVIEW

File: pkg/lib/cockpit-components-plot.jsx [hunk 1]
```diff
@@ -404,8 +404,10 @@ export const SvgPlot = ({ title, config, plot_state, plot_id, className }) => {
             if (x < m_left) x = m_left;
             if (x > w - m_right) x = w - m_right;
             setSelection({
-                start: selection.start, stop: x,
-                left: Math.min(selection.start, x), right: Math.max(selection.start, x)
+                start: selection.start,
+                stop: x,
+                left: Math.min(selection.start, x),
+                right: Math.max(selection.start, x)
```
Comment (cockpituous) [inline pkg/lib/cockpit-components-plot.jsx:410]: These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17890-20221107-162851-fa6168ae-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/iscsi-panel.jsx [hunk 2]
```diff
@@ -226,7 +226,11 @@ export class IscsiPanel extends React.Component {
                 );
 
             return {
-                client, actions, kind: "array", name: session.data.target_name || "", key: path,
+                client,
+                actions,
+                kind: "array",
+                name: session.data.target_name || "",
```
Comment (cockpituous) [inline pkg/storaged/iscsi-panel.jsx:232]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17890-20221107-162851-fa6168ae-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/stratis-panel.jsx [hunk 3]
```diff
@@ -51,18 +51,27 @@ function stratis_pool_row(client, path) {
...
     return {
-        client, actions: action, name: uuid, key: uuid, truncate_name: false,
-        detail: _("Locked encrypted Stratis pool"), go: () => cockpit.location.go(["pool", uuid])
+        client,
+        actions: action,
+        name: uuid,
+        key: uuid,
+        truncate_name: false,
+        detail: _("Locked encrypted Stratis pool"),
+        go: () => cockpit.location.go(["pool", uuid])
```
Comment (cockpituous) [inline pkg/storaged/stratis-panel.jsx:74]: These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17890-20221107-162851-fa6168ae-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/vdos-panel.jsx [hunk 4]
```diff
@@ -28,7 +28,11 @@ const _ = cockpit.gettext;
 function vdo_row(client, vdo) {
     const block = client.slashdevs_block[vdo.dev];
     return {
-        client, key: vdo.name, kind: "array", name: vdo.name, devname: vdo.dev,
+        client,
+        key: vdo.name,
+        kind: "array",
+        name: vdo.name,
+        devname: vdo.dev,
```
Comment (cockpituous) [inline pkg/storaged/vdos-panel.jsx:35]: These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17890-20221107-162851-fa6168ae-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit.js [hunk 5]
```diff
@@ -2972,7 +2973,8 @@ function factory() {
             },
             call: {
  value: function(name, args, options) { return client.call(path, iface, name, args, options) },
-                      enumerable: false, writable: false
+                      enumerable: false,
+writable: false
```
Comment (martinpitt) [inline pkg/lib/cockpit.js:2977]: And this is even worse

**Review by martinpitt — CHANGES_REQUESTED** [review]

File: pkg/lib/cockpit.js [hunk 6]
```diff
@@ -2963,7 +2963,8 @@ function factory() {
             iface: { value: iface, enumerable: false, writable: false },
             valid: { get: function() { return valid }, enumerable: false },
             wait: {
- enumerable: false, writable: false,
+ enumerable: false,
+writable: false,
```
[the remark this answers is not in the document]
Comment (jelly) [reply pkg/lib/cockpit.js:2967]: Ugh... `eslint --fix` does not fix super well

**Review by martinpitt — APPROVED** [review]
Cheers!

File: pkg/lib/cockpit-components-plot.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-plot.jsx:410]: These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17890-20221108-090107-5c6c50b0-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/iscsi-panel.jsx (hunk 2 above)
Comment (cockpituous) [inline pkg/storaged/iscsi-panel.jsx:232]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17890-20221108-090107-5c6c50b0-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/stratis-panel.jsx (hunk 3 above)
Comment (cockpituous) [inline pkg/storaged/stratis-panel.jsx:74]: These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17890-20221108-090107-5c6c50b0-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/vdos-panel.jsx (hunk 4 above)
Comment (cockpituous) [inline pkg/storaged/vdos-panel.jsx:35]: These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17890-20221108-090107-5c6c50b0-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Pull request merged**